build/config.props

```diff
@@ -37,8 +37,8 @@
     <!--TODO: remove temporary workaround of using LockSDKVersion. Tracking issue: https://github.com/NuGet/Home/issues/9712 -->
     <LockSDKVersion>true</LockSDKVersion>
     <OverrideCliBranchForTesting Condition="'$(LockSDKVersion)' == 'false'"></OverrideCliBranchForTesting>
-    <OverrideCliBranchForTesting Condition="'$(LockSDKVersion)' == 'true'">master 5.0.100-preview.7.20319.6</OverrideCliBranchForTesting>
-    <CliVersionForBuilding>"master 5.0.100-preview.4.20258.7"</CliVersionForBuilding>
+    <OverrideCliBranchForTesting Condition="'$(LockSDKVersion)' == 'true'">master 5.0.100</OverrideCliBranchForTesting>
```

Comment:
I think we can remove `OverrideCliBranchForTesting` and `LockSDKVersion` properties and only rely on `CliVersionForBuilding` and `CliBranchForTesting` properties. This should also fix https://github.com/NuGet/Home/issues/9712.

Reply:
As this PR is needed to unblock CI builds for the whole team, we'll do the suggestion in another PR.